packages/sdk/wallets/wallet-local/src/signing.test.ts

```diff
@@ -199,6 +212,8 @@ describe('Transaction Utils', () => {
         chainId,
         gas,
         feeCurrency: i % 3 === 0 ? feeCurrency : undefined,
+        // TODO: No idea which modulo it should be. Can we document this?
+        maxFeeInFeeCurrency: i % 3 === 1 ? gasPrice : undefined,
```

Comment:
yeah i didnt understand this either when i started changing things around.

Reply:
i would do module 6 or 9 so that there are some feecurrency with and without. but maxFeeIn always is with feeCurrency

Reply:
🤪